__tests__/app.test.js

```diff
@@ -142,6 +142,63 @@ describe("GET /api/articles errors", () => {
 
 })
 
+describe('GET /api/articles/:article_id/comments', () => {
+test('200: Responds with an ampty array if the id exists, but there are no comments', () => {
+    return request(app)
+    .get("/api/articles/2/comments")
+    .expect(200)
+    .then(({ body }) => {
+        expect(body.comments).toEqual([])
+    })
+    });
+test('Responds with a correct array of comments when given a valid article_id', () => {
+    return request(app)
+    .get('/api/articles/6/comments')
+    .expect(200)
+    .then(({ body }) => {
+        const { comments } = body
+        expect(comments).toEqual([{
+            comment_id: 16 ,
+            body: "This is a bad article name",
+            votes: 1,
+            author: "butter_bridge",
+            article_id: 6,
+            created_at: "2020-10-11T15:23:00.000Z"
+          }])
```

Comment:
Would be better to check the shape of the comments rather than the actual comments for a certain article